global-privacy-control/tests.json

```diff
@@ -0,0 +1,129 @@
+{
+    "gpcHeader": {
+        "name": "Tests related to the GPC request header",
+        "desc": "Check if `Sec-GPC: 1` is correctly sent with all requests",
+        "tests": [
```

Comment:
FWIW I don't have much familiarity with GPC itself so not opining on the accuracy of these tests.  Not sure if you want a separate reviewer for that purpose?

Reply:
Yeah, good call, I’ll ping Dave 👍